hw/ip/i2c/README.md

```diff
@@ -396,27 +396,27 @@ After reset, the initialization of the I2C HWIP primarily consists of four steps
 Of the four initialization steps, the timing parameter initialization is the most involved.  With so many timing parameters, it is essential to have dedicated device interface functions (DIFs) to determine appropriate values for the 10 timing parameters.
 
 The values of these parameters will depend primarily on three bus details:
-- The speed mode of the slowest device on the bus: standard mode (100 kbaud), fast mode (400 kbaud) or fast-mode plus (1 Mbaud).
+- The speed mode of the slowest device on the bus: Standard-mode (100 kbaud), Fast-mode (400 kbaud) or Fast-mode Plus (1 Mbaud).
 - The input clock period, t<sub>clk</sub> in ns.
 - The expected signal rise time, t<sub>r</sub>, in ns.
-   - This is not a firmware-controlled parameter.
-Rather, it is a function of the capacitance and physical design of the bus.
-The specification provides detailed guidelines on how to manage capacitance in an I2C system:
-   - Section 5.2 of the I2C specification indicates that Fast-mode plus devices may operate at reduced clock speeds if the bus capacitance drives signal rise times (t<sub>r</sub>) outside the nominal 120ns limit.
-Excess capacitance can also be compensated for by reducing the size of the bus pullup resistor, so long as the total open-drain current does not exceed 20mA for fast-mode plus devices (as described in section 7.1 of the I2C specification).
-However the specification places a hard limit on rise times capping them at 1000ns.
-    - If there are standard- or fast-mode target devices on the bus, the specified open-drain current limit is reduced to 3mA (section 7.1), thus further restricting the minimum value of the pull-up resistor.
-    - In fast-mode bus designs, where the total line capacitance exceeds 200pF, the specification recommends replacing the pull-up resistor with an active current source, supplying 3mA or less (section 5.1).
-Regardless of the physical construction of the bus, the rise time (t<sub>r</sub>) is a system dependent, parameter that needs to be made known to firmware for I2C initialization.
+  - This is not a firmware-controlled parameter.
+  Rather, it is a function of the capacitance and physical design of the bus.
+  The specification provides detailed guidelines on how to manage capacitance in an I2C system:
+  - Section 5.2 of the I2C specification indicates that Fast-mode Plus devices may operate at reduced clock speeds if the bus capacitance drives signal rise times (t<sub>r</sub>) outside the nominal 120ns limit.
+  Excess capacitance can also be compensated for by reducing the size of the bus pullup resistor, so long as the total open-drain current does not exceed 20mA for Fast-Mode plus devices (as described in section 7.1 of the I2C specification).
```

Comment:
`Fast-mode Plus` or `Fm+` (to match the spec)

Reply:
Thanks, should be resolved now.